test/app/views/application/content_for.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render ContentForComponent do |comp| %>
+  <p>Have a great day.</p>
```

Comment:
I wonder if we should support this "mixed" syntax, with some named blocks used in combination with an implicit block. It could be confusing to have these bits in a particular order in the template, only to have the component render them in a different order.

Reply:
interesting. It certainly makes testing a PITA because of the whitespace. 

Would the expectation be that if you supply content areas then we no longer capture the rest of the block as the 'content' (or perhaps renamed to 'body')